code/__HELPERS/files.dm

```diff
@@ -73,6 +73,22 @@
 				warning("Skipping map [potential] due to  it being too hot outside. Ideal temp is below 40F, found [temperature].")
 				binary = null
 				continue
+//Lamprey currently has a tendency to crash for no real reason, uncomment this when it stops being broken
+/*
+		if(potential == "Lamprey Station/") //Available if the station is wrecked enough
```

Comment:
delete all this

Reply:
This will be uncommented when Lamprey gets fixed, it's just convenience